search/includes/classes/class-healthjob.php

```diff
@@ -240,6 +252,8 @@ public function check_document_count_health() {
 
 			$this->process_document_count_health_results( $post_results );
 		}
+
+		$this->remove_healthcheck_lock();
```

Comment:
Does it make sense to use `try…finally` here? For example, if an unexpected exception happens?

Reply:
@sjinks I figured that IF something were to unexpectedly happen where the lock couldn't be removed, it would eventually expire after 5 minutes.

Reply:
Oh, right, `set_transient()`